Update rotating drum example

blaisb opened the pull request:
# Description of the problem

- The large scale rotating drum example contained poorly set parameters and did not run anymore.

# Description of the solution

- Fixed the paremeters and split the example into two parts to load the particles correctly
- Update documentation

acdaigneault commented on doc/source/examples/dem/rotating-drum/rotating-drum.rst:
```diff
@@ -167,14 +166,14 @@ This simulation can be launched in two steps. First the particles need to be loa
 
 .. warning::
-	Loading the particles in this example requires approximately 50 minutes on 8 cores. Simulating the motion of the particles requires an additional 8 hours. This high computational cost is because of the large number of particles and the long duration of the simulation.
+	This example needs a simulation time of approximately 12 hours 8 cores. 
```
```suggestion
	This example needs a simulation time of approximately 12 hours on 8 cores. 
```

blaisb replied:
That's mistake. I actually commited the two new prm files when I did the refactoring of the dim of the solver. So it confused the history. That's why the prm are missing from the commits also.

acdaigneault reviewed (commented):
Are the modified .prm missing? You say you changed the parameters, but here we only see some clean up in the doc

blaisb commented:
> Are the modified .prm missing? You say you changed the parameters, but here we only see some clean up in the doc + new video.

The PRM are there, but I added them by mistake when I refactored the dimensionality. The new PRM are actually already there in the master version of Lethe.

voferreira commented on doc/source/examples/dem/rotating-drum/rotating-drum.rst:
```diff
@@ -33,7 +32,7 @@ Parameter File
-In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius and half-length, respectively. Therefore, the specified grid arguments create a cylinder with a diameter of 0.24 m and a length of 0.36 m. The grid is refined 4 times to reach the desired cell size to particle diameter ratio (see packing in ball example for more details). The ``expand particle- ...
+In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius and half-length, respectively. Therefore, the specified grid arguments create a cylinder with a diameter of 0.24 m and a length of 0.36 m. The grid is refined 4 times to reach the desired cell size to particle diameter ratio (see packing in ball example for more details). The ``expand particle- ...
 
 .. code-block:: text
```
The `subsection mesh` in the example .prm file is:

subsection mesh
  set type                                = dealii
  set grid type                           = subdivided_cylinder
  set grid arguments                      = 4: 0.12:0.18
  set initial refinement                  = 4
  set expand particle-wall contact search = true
end

voferreira replied:
The young modulus is 1e7 instead of 1e8 (walls and particles)

voferreira replied:
The insertion info is outdated in the `rotating-drum.prm` file (I know it doesn't play a role since `set inserted number of particles at each time step = 0`, but still)

voferreira replied:
Load balancing parameters do not match on both prm files. That is fine, but it is not clear in the example what is in the load and what is in the rotating drum file. Not a big deal, but I would maybe point that info out through the example.

voferreira replied:
The `subsection simulation control` in the .prm file of the example is:

subsection simulation control
  set time step         = 1e-5
  set time end          = 12
  set log frequency     = 1000
  set output frequency  = 1000
  set output boundaries = false
  set output path       = ./output/
end

voferreira reviewed (approved):
Just some small suggestion. I tested both prm files on my machine (first time-steps only) and they are working.

blaisb commented:
All comments have been addressed @voferreira

voferreira commented:
> All comments have been addressed @voferreira 

Nice!! On my side, it is ready to go.

blaisb commented:
Will merge after CI then
:)
